### ISSUE TITLE
[BUG] runner is reporting old Node.js version.

### ISSUE THREAD

**Issue by nsalfos** [tj-actions/changed-files#678]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Does this issue exist in the latest version?

- [X] I'm using the latest release

### Describe the bug?

When running a workflo with this actions is now reporting:
`
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: tj-actions/glob, tj-actions/globShow less
`

### To Reproduce

Just try to run the actions against any repo with changed files. 

### What OS are you seeing the problem on?

ubuntu-latest or ubuntu-20.04

### Expected behavior?

the file is being listed anyways, but not sure if it is not bringing any errors with it.

### Relevant log output

```shell
`
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: tj-actions/glob, tj-actions/globShow less
`
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by heathdutton** [comment]
Confirmed:
```
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: tj-actions/glob, tj-actions/glob
```

**Comment by jackton1** [comment]
This has been resolved in the latest patch release [v32.0.1](https://github.com/tj-actions/changed-files/releases/tag/v32.0.1) and is also updated in the current major release [v32](https://github.com/tj-actions/changed-files/releases/tag/v32). Thanks once again for you keen observation.
